portfolio/src/main/webapp/index.html

```diff
@@ -24,10 +24,18 @@ <h2>Background</h2>
          that yes, New Jersey is home to the best bagels.    
       <p>Don't believe me? You simply HAVE to come to New Jersey and try them.  It should be the 
          number one location on your bucket list.   
-
       <p>Click here for a random famous New Jerseyian:</p>
       <button onclick="addRandomNJPerson()">NJ Rocks!</button>
       <div id="NJ-person-container"><p id="NJ-person-name"><p id="NJ-person-descrip"></div>
+
+      <h2>More about me</h2>
+      <h2>Contact Me</h2>
+      <ul>
+        <li>Work email: isangimino@google.com
+        <li>Personal email: isabelsang18@gmail.com
+        <li><a href="https://github.com/isabelsang">My Github</a>
+        <li><a href="https://www.linkedin.com/in/isabel-sangimino-b56973190">My Linkedin</a>
```

Comment:
Nit: It's extremely common to leave out closing li tags, but I would include them since it makes behavior more predictable.

```suggestion
        <li>Work email: isangimino@google.com</li>
        <li>Personal email: isabelsang18@gmail.com</li>
        <li><a href="https://github.com/isabelsang">My Github</a></li>
        <li><a href="https://www.linkedin.com/in/isabel-sangimino-b56973190">My Linkedin</a></li>
```